src/clock.rs

```diff
@@ -39,7 +38,16 @@ impl Instant {
             (
                 InstantKind::Virtual { nanoseconds },
                 InstantKind::Virtual { nanoseconds: earlier },
-            ) => Duration::from_nanos(nanoseconds.saturating_sub(earlier)),
+            ) => {
+                // It is possible for second to overflow because u64::MAX < (u128::MAX / 1e9).
+                let seconds = u64::try_from(
+                    nanoseconds.saturating_sub(earlier).saturating_div(1_000_000_000),
+                )
+                .unwrap();
```

Comment:
Knowing that it is possible to overflow, is it better to unwrap and panic, or manually handle the ``try_from`` error by setting``seconds`` to ``u64::MAX``?